src/components/Metrics/index.js

```diff
@@ -0,0 +1,100 @@
+import { RichTypography } from "@commons-ui/core";
+import { IconButton, Grid, useMediaQuery } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import useStyles from "./useStyles";
+
+import DataVisualCard from "@/pesayetu/components/DataVisualCard";
+import Section from "@/pesayetu/components/Section";
+
+function Metrics({ items, sectionTitle, ...props }) {
+  const classes = useStyles(props);
+  const theme = useTheme();
+  const isTablet = useMediaQuery(theme.breakpoints.only("md"));
+  if (!items?.length) {
+    return null;
+  }
+  return (
+    <div className={classes.root}>
+      <Section classes={{ root: classes.section }}>
+        <RichTypography variant="h2" className={classes.sectionTitle}>
+          {sectionTitle}
+        </RichTypography>
+        {items?.map((item, index) => (
+          <Grid
+            key={item.title}
+            container
+            direction={index === 0 ? "row" : "row-reverse"}
```

Comment:
```suggestion
            direction={index % 2 === 0 ? "row" : "row-reverse"}
```
This will keep alternating row, row-reverse even if we had 100 rows